tools/scripts/start-local-registry.ts

```diff
@@ -16,6 +16,7 @@ export default async () => {
     verbose: true,
   });
 
+  // is is also possible to use nx release to publish the packages to the local registry
```

Comment:
I just left this comment also to let you know that `tools/scripts/publish.mjs` could be replaced and all the `publish` targets removed to instead use `nx release publish --first-release`.
Your choice.

Reply:
That is an attractive option, will look into that at some point :+1: